routes/randomActivity.js

```diff
@@ -8,6 +8,7 @@ const app = express();
 app.get("/random", async function (request, response) {
   const aggregates = await ActivityModel.aggregate([{ $sample: { size: 1 } }]);
 
+  // Transform the aggregate array to an object
   const activity = {
```

Comment:
```suggestion
  // Because we requested a sample size of 1, so we extract our response as the first item of the aggregate response array
  const activity = {
```

Reply:
updated